applications/Cere_Turnkey_Private_Blockchain_Network.md

```diff
@@ -2,7 +2,7 @@
 
 * **Project:** Turn-key Private Standalone Blockchain Network
 * **Proposer:** [Cere-Network](https://github.com/Cerebellum-Network)
-* **Payment Address:** 1BbyAGddobdPgNo9c63xsRnwK5hen8pV7F 
+* **Payment Address:** 3A79EuVJQzZPZSYZ4LDMCxqNUy95xJtFng
```

Comment:
Is there a reason for this change?

Reply:
Hi @mmagician, it was just a preference to use a cold wallet storage vs a custodial one. Let us know if this is an issue.